src/main/java/com/google/sps/servlets/getArticlesServlet.java

```diff
@@ -81,7 +82,8 @@ public void doPut(HttpServletRequest request, HttpServletResponse response) thro
     // Deletes queries from last doPut so the datastore results can be updated.
     Query query = new Query("Article");
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-    deleteResultsOfQueryFromDatastore(query, datastore);
+    String s = new String("Article");
```

Comment:
Give it a meaningful variable name. Also make it available to the entire class so that it is reusable.

Alternatively, you can declare an enum in the utility class, so all servlets will have access to that enum.

Reply:
Fixed: Gave it kind and made it available to the entire class.